libraries/AP_TemperatureSensor/AP_TemperatureSensor_Analog.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_TemperatureSensor_Analog.h"
+
+#if AP_TEMPERATURE_SENSOR_ANALOG
```

Comment:
```suggestion
#if AP_TEMPERATURE_SENSOR_ANALOG_ENABLED
```

Reply:
I second this. Needs enabled in the define name

Reply:
Done